controlplane/pkg/plugins/utils.go

```diff
@@ -97,3 +101,26 @@ func registerPlugin(name, endpoint, registry string, capabilities []plugins.Plug
 
 	return err
 }
+
+func monitorNSMD(name string, endpoint string, registry string, capabilities []plugins.PluginCapability) {
+	registered := true
+	probeChecker := probes.NewNsmdProbeChecker()
+	for ; ; <-time.After(livenessCheckPeriod) {
+		nsmdAlive, err := probeChecker.Alive()
```

Comment:
I think we should use probeChecker.Ready() here

Reply:
- [ ] check probeChecker.Ready() before registration